var/spack/repos/builtin/packages/scallop/package.py

```diff
@@ -17,10 +17,7 @@ class Scallop(AutotoolsPackage):
     version('0.10.3', sha256='04eb3ab27ed8c7ae38e1780d6b2af16b6a2c01807ffafd59e819d33bfeff58a0')
 
     depends_on('clp')
-
-    # TODO: replace this with an explicit list of components of Boost,
-    # for instance depends_on('boost +filesystem')
-    # See https://github.com/spack/spack/pull/22303 for reference
+    # Fixme: There does not seem to be any dependency on boost, please consider removing
```

Comment:
Both versions of scallop depend on `boost/icl` (see src/interval_map.h). icl is header-only.

Reply:
I'll add a suggested change below for this.

Reply:
```suggestion
```